common/resource/resource_test.go

```diff
@@ -25,77 +25,12 @@ package resource
 import (
 	"testing"
 
-	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
-
-	"github.com/uber/cadence/common/cache"
-	"github.com/uber/cadence/common/cluster"
-	"github.com/uber/cadence/common/dynamicconfig"
-	"github.com/uber/cadence/common/isolationgroup"
-	"github.com/uber/cadence/common/log/loggerimpl"
-	"github.com/uber/cadence/common/metrics"
-	"github.com/uber/cadence/common/persistence"
-	"github.com/uber/cadence/common/service"
 )
 
-func TestEnsureIsolationGroupImpl(t *testing.T) {
```

Comment:
Removing this because the logic for determining this is moved to the IsolationGroup package